momentGW/ints.py

```diff
@@ -43,15 +43,14 @@ def __init__(
         compression_tol=1e-10,
         store_full=False,
     ):
-        self.verbose = with_df.mol.verbose
-        self.stdout = with_df.mol.stdout
-
         self.with_df = with_df
         self.mo_coeff = mo_coeff
         self.mo_occ = mo_occ
         self.compression = compression
         self.compression_tol = compression_tol
         self.store_full = store_full
+        self.verbose = with_df.mol.verbose
+        self.stdout = with_df.mol.stdout
```

Comment:
How come the `verbose` and `stdout` lines moved here? They're not really parameters so please revert this to:
```
...
):
        self.verbose = with_df.mol.verbose
        self.stdout = with_df.mol.stdout

        self.with_df = with_df
...
```
with the blank line there also